apps/frontend/src/lib/storage/localStorage/user/oauth.ts

```diff
@@ -0,0 +1,38 @@
+import { AccessToken, AccessTokenSchema } from "@types";
+import { getUserAndSession } from ".";
+import { saveBackupAndUpdateStorage } from "../utils";
+import { createOAuthBackup } from "@/lib/backup";
+
+export const saveOAuthAccessToken = async (
+  app: string,
+  token: AccessToken
+): Promise<void> => {
+  const { user, session } = await getUserAndSession();
+
+  const oauthBackup = createOAuthBackup({
+    email: user.email,
+    password: session.backupMasterPassword,
+    oauthData: {
+      app,
+      token,
```

Comment:
Just to check -- it's possible to update this in the future for scope? Or if the scope changes should we just overwrite the existing token?

Reply:
yeah we can just overwrite